packages/polymath-js/src/contracts/Contract.js

```diff
@@ -200,7 +200,7 @@ export default class Contract {
     const params = {
       ...preParams,
       gas,
-      gasPrice: 5000000000,
+      gasPrice: 10000000000,
```

Comment:
What are the implications of this change in production?

Reply:
People will be charged double gas price for their transactions but they will mine faster. Most transactions are under 10 cents normally anyway. Maybe it's something we should consult with the product team first?

Reply:
```suggestion
      gasPrice: 5000000000,
```